spring-cloud-gcp-pubsub-stream-binder/src/test/java/com/google/cloud/spring/stream/binder/pubsub/PubSubMessageChannelBinderTests.java

```diff
@@ -148,4 +172,67 @@ public void consumerMaxFetchPropertyPropagatesToMessageSource() {
 				});
 	}
 
+	@Test
+	public void testProducerAndConsumerCustomizers() {
+		baseContext.withUserConfiguration(PubSubBinderTestConfig.class)
+				.withPropertyValues("spring.cloud.stream.bindings.input.group=testGroup")
```

Comment:
This test relies on a lot of internal Spring Cloud Stream implementation details. Would it be easier to retrieve the adapter/message source by name? Or customize ACK method and verify that instead of verifying the name?

Reply:
It might be possible, but I simply followed the test design for Kafka binder.